### ISSUE TITLE
[JIT] not working with @craco/craco

### ISSUE THREAD

**Issue by createdbymahmood** [tailwindlabs/tailwindcss#3998]
### What version of Tailwind CSS are you using?

2.1.0

### What version of Node.js are you using?

12.20.0

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### Reproduction repository

https://github.com/createdbymahmood/craco-tailwindcss-problem

I'm using TailwindCSS with craco, to support "css modules apply syntax", in addition, I want to use "Just-in-time", it does not work as expected, styles are not getting compiled fine. only tailwind base styles (normalizers) are available to use now.

**Comment by devpeerapong** [comment]
I think JIT [doesn't support PostCSS 7](https://github.com/tailwindlabs/tailwindcss-jit/issues/19#issuecomment-799757696)

**Comment by sveisvei** [comment]
Indeed, the provided reproduction repository does have the Postcss 7: 
https://github.com/createdbymahmood/craco-tailwindcss-problem/blob/main/package-lock.json#L2029

**Comment by bradlc** [comment]
Hey @createdbymahmood, two things:

1. You need to upgrade tailwindcss to `2.1.2` – there was a bug relating to PostCSS 7 which was fixed in #4078:
  ```
  npm uninstall @tailwindcss/postcss7-compat tailwindcss
  npm install tailwindcss@npm:@tailwindcss/postcss7-compat
  ```
2. You need to [specify your template paths](https://tailwindcss.com/docs/just-in-time-mode#enabling-jit-mode) in the `purge` option of your `tailwind.config.js` file. Without this the JIT engine does not know which classes to generate. For example:
  ```
  module.exports = {
    purge: ['./src/**/*.js'],
    // ...
  }
  ```

I have opened [a pull request](https://github.com/createdbymahmood/craco-tailwindcss-problem/pull/1) on your reproduction repository and confirmed that it's now working 👍 Any other issues let us know!
